packages/datagateway-search/src/table/datasetSearchTable.component.tsx

```diff
@@ -123,6 +206,38 @@ const DatasetSearchTable = (
     />
   );
 
+  const fetchFacilityCycles = React.useCallback(() => {
```

Comment:
This function is repeated in all three tables - but since this will eventually be refactored with the `react-query` changes it's fine for now